sys-apps/razercfg/razercfg-0.42-r1.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```

Comment:
Please drop python3_7 (it's already disabled system-wide, so we can remove it entirely)

Reply:
done